_conferences/codemotion-madrid-2019.md

```diff
@@ -0,0 +1,10 @@
+name: "Codemotion Madrid"
```

Comment:
We omit the location from the name because it will be displayed using the actual location below

Reply:
FIxed